src/pages/events/[id]/movie.tsx

```
@@ -15,198 +15,198 @@ import { toast } from "react-toastify";
-            onClose={closeDialog}
-            onConfirm={handleConfirm}
-            title="動画を削除しますか？"
-            text="この操作は取り消せません。"
-            confirmText="削除する"
-          />
-        </div>
-      </DefaultLayout>
-    </>
-  );
+	const router = useRouter();
+	const { id } = router?.query;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)